--- a/my_machine.h	2025-01-18 11:33:00.351266809 -0800
+++ b/my_machine.h	2025-01-18 11:33:38.523395486 -0800
@@ -21,7 +21,7 @@
 
 // NOTE: Only one board may be enabled!
 // If none is enabled pin mappings from generic_map.h will be used.
-//#define BOARD_PICO_CNC
+#define BOARD_PICO_CNC
 //#define BOARD_RP23U5XBB
 //#define BOARD_PICOBOB
 //#define BOARD_PICOBOB_G540
@@ -37,9 +37,9 @@
 // Configuration
 // Uncomment to enable.
 
-#ifndef USB_SERIAL_CDC
-#define USB_SERIAL_CDC          1 // Serial communication via native USB.
-#endif
+//#ifndef USB_SERIAL_CDC
+//#define USB_SERIAL_CDC          1 // Serial communication via native USB.
+//#endif
 //#define BLUETOOTH_ENABLE        2 // Set to 2 for HC-05 module, enable in CMakeLists.txt if for Pico W Bluetooth.
 // Spindle selection:
 // Up to four specific spindle drivers can be instantiated at a time
